### ISSUE TITLE
"Sort by" and "Appearance" settings for Favourites, People, Rooms and Low Priority do not persist between devices

### ISSUE THREAD

**Issue by DMRobertson** [vector-im/element-web#22902]
### Steps to reproduce

1. Log in to a new instance of Element.
2. "Favourites" are set to "Sort by A-Z" with no "Appearance" options checked.
3. Set "Favourites" to "Sort by Activity" and  "Show rooms with unread messages first".
4. Open another instance of Element.
  - Either log out and log in, or e.g. use Element in a browser rather than Element-desktop.

#### What did you expect?

"Favourites" is still set to "Sort by Activity" and  "Show rooms with unread messages first".

More generally, I would expect this piece of configuration to be synced across _all_ element-web instances via global account data. Possibly across Element mobile clients too.

#### What happened instead?

"Favourites" on the new device are set to "Sort by A-Z" with no "Appearance" options checked, i.e. their default state.

### Operating system

Fedora 36

### Application version

Element version: 1.11.0 Olm version: 3.2.8

### How did you install the app?

Flatpak, https://flathub.org/apps/details/im.riot.Riot . I know this isn't supported but am reporting for completeness.

### Homeserver

matrix.org

### Will you send logs?

Yes

**Comment by nadonomy** [comment]
This tweak sounds good to me - I'm apprehensive about syncing sorting settings cross-platform from e.g. desktop to mobile, as we can't guarantee the apps will have the same layout options.

But, saving users from the busywork of needing to re-configure across multiple browser/desktop sessions sounds like a win.

**Comment by SISheogorath** [comment]
Yep, regular papercut issue. Always takes a minute to setup and happens multiple times a year due to whatever happens to session management in the client and requiring a relogging.
